### ISSUE TITLE
cmd-` as a keyboard shortcut on macOS hijacks the OS shortcut for switching between windows of an app

### ISSUE THREAD

**Issue by ara4n** [vector-im/riot-web#14381]
this is pretty bad form; it's almost as bad as overloading cmd-tab

**Comment by tedd4u** [comment]
Backtick to preferences shortcut appears to be defined at line 204 of this file:

https://github.com/matrix-org/matrix-react-sdk/blob/8a0ee2fd34adfd5d7cdd03e625284458892f427a/src/accessibility/KeyboardShortcuts.tsx#L204

**Comment by SimonBrandner** [comment]
No, it's actually defined [here](https://github.com/matrix-org/matrix-react-sdk/blob/cd39474d26dbc5d6b5dc548ea7c0e0aaa6b0e21e/src/KeyBindingsDefaults.ts#L340). The file you linked just defines what is shown in the dialog that lists the keybindings. In the future, that dialog should be replaced by one that reads the keybindings directly from the file where they are defined
